### ISSUE TITLE
Expiring steps in automation

### ISSUE THREAD

**Issue by Siiffloor** [Simplero/Roadmap#449]
Expiring/inactive steps in automation

I would like a setting to let a step in an automation expire/go inactive.

Case:
I am running a webinar.
And some steps become irrelevant if the participants sign up later in the process.
As the automations works now. I manually have to add go to steps on the day a step becomes irrelevant

Example
Step 1: Templated email (welcome email)
Step 2: Wait for a specific date
Step 3: Templated email (The webinar is tomorrow)
Step 4: Wait for 1 hour then wait until "day of the month" 8:00am
Step 5: Templated email (The webinar start in 10 minutes)

Here it would be nice to be able to automatically let step 3, 4 and 5 expire after a date and time of my choice on each step, instead of having reminders in my calendar to add/change go to steps in the calendar every morning of a campaign.

It would just be great to know that once this is set up, I don't have to spend more energy on that part of the launch.

This would also be great when building a campaign, where you are offering early bird prices in the first period of a campaign, to everyone subscribing to the list up in that period, but not after a specific date.

Thanks a lot

**Comment by moso1975** [comment]
Agreed. Would be a great feature! Today one must manually set up every "Go to" in the right time - time consuming and error-prone.
